iothub/device/tests/Authentication/DeviceAuthenticationWithTokenRefreshTests.cs

```diff
@@ -154,9 +154,10 @@ public async Task DeviceAuthenticationWithSakRefresh_SharedAccessKeyConnectionSt
             Assert.IsNotNull(iotHubConnectionCredentials.SasTokenRefresher);
             Assert.IsInstanceOfType(iotHubConnectionCredentials.SasTokenRefresher, typeof(DeviceAuthenticationWithSakRefresh));
 
+            var auth = (IConnectionCredentials)iotHubConnectionCredentials;
```

Comment:
GetPasswordAsync is implemented on IotHubConnectionCredentials through explicit interface implementation, so it can only be accessed through IConnectionCredentials. Defining iotHubConnectionCredentials as IConnectionCredentials works too.

Reply:
I see.

Why implement it through explicit interface implementation?

Reply:
Explicit implementation is required because IClientCrendetials.GetPasswordAsync is now internal whereas IotHubConnectionCredential (which implements IConnectionCredentials) is public.